tv_apps/dashboard/js/dashboard.js

```diff
@@ -58,6 +73,7 @@
       var direction = document.body.dataset.activeDirection;
       this.widgets[direction] && this.widgets[direction].toggleExpand(false);
       document.body.dataset.activeDirection = '';
+      document.activeElement.blur();
```

Comment:
We should keep focus at dashboard after we blur somebody. So, please add something like: `mainSection.focus()`.

We had met an issue at smart-home. While we only blur someone and keep focus at smart-home, other apps under smart-home may have opportunity to steal focus.